optimizely/marketing-website
# Upgrade grunt-sass

- pull grunt-sass directly from GitHub
- pull css-smart-grid from forked optimizely repo

current error:
```
>> You may not @extend an outer selector from within @media.
>> You may only @extend selectors within the same directive.
>> From "@extend .clearfix" on line 337 of website-guts/assets/css/helper-classes.scss
```

reference 
http://thesassway.com/intermediate/understanding-placeholder-selectors#limitations

these changes are confusing because of the `@extend` error. If you want to learn more read 
http://www.sitepoint.com/cross-media-query-extend-sass/
but I basically copy/pasted this code.

#### To Test

- Check all pages with changes in different browser sizes
- run `grunt build` to see if CSS is minified
- check staging https://www.optimizelystaging.com/dfoxpowell/upgrade-grunt-sass/
